kvrocks.conf

```diff
@@ -597,6 +597,30 @@ rocksdb.enable_blob_garbage_collection yes
 # Default: 25
 rocksdb.blob_garbage_collection_age_cutoff 25
 
+
+# The purpose of following three options are to dynamically adjust the upper limit of
+# the data that each layer can store according to the size of the different 
+# layers of the LSM. Enabling this option will bring some improvements in
+# deletion efficiency and space amplification, but it will lose a certain 
+# amount of read performance.
+# If you want know more detail about Levels' Target Size, you can read RocksDB wiki: 
+# https://github.com/facebook/rocksdb/wiki/Leveled-Compaction#levels-target-size
+#
+# Default: no
+rocksdb.level_compaction_dynamic_level_bytes no
+
+# The total file size of level-1 sst.
+#
+# Default: 256 M
+rocksdb.max_bytes_for_level_base 268435456
```

Comment:
I think `MB` is much readable, even lose flexibility, and currently, we usually use `MB` too, @git-hulk WDYT?

Reply:
It should be fine to keep the same as RocksDB since we expected only a few users would adjust those options.

Reply:
@git-hulk @ShooterIT I'd updated max_bytes_for_level_base unit bytes to MiB.